libs/components/forms/src/lib/modules/file-attachment/file-drop/file-drop.component.spec.ts

```diff
@@ -1256,15 +1256,21 @@ describe('File drop component', () => {
     );
   });
 
-  it('should set aria-describedby for the link input', () => {
+  it('should set hint text for the link input', async () => {
+    const linkUploadHintText = 'Hello, world!';
+
     componentInstance.allowLinks = true;
+    componentInstance.linkUploadHintText = linkUploadHintText;
     fixture.detectChanges();
 
     const linkInput = getLinkInput();
 
+    const ariaDescribedby =
+      linkInput.nativeElement.attributes.getNamedItem('aria-describedby').value;
+
     expect(
-      linkInput.nativeElement.attributes.getNamedItem('aria-describedby').value,
-    ).toBe('MOCK_ID_4');
+      document.getElementById(ariaDescribedby)?.textContent?.trim(),
```

Comment:
I wish we could use the `SkyInputBoxHarness` here, but I was getting a build error about a "circular reference".